src/openrct2/ride/TrainManager.h

```diff
@@ -0,0 +1,76 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2021 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+#pragma once
+#include <cstdint>
+#include <list>
+
+struct Vehicle;
+
+namespace TrainManager
+{
+    class TrainView
```

Comment:
It would be probably also good to comment that the View is only returning vehicle heads, this might confuse someone trying to iterate all vehicles.